tsdb/query_executor_test.go

```diff
@@ -41,10 +41,10 @@ func TestWritePointsAndExecuteQuery(t *testing.T) {
 		t.Fatalf(err.Error())
 	}
 
-	got := executeAndGetJSON("select * from cpu", executor)
+	got := executeAndGetJSON("SELECT * FROM cpu GROUP BY *", executor)
```

Comment:
Here again, can we add one test that does not do the `GROUP BY`? Or is that functionality not ready yet?